src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingActualsResponseMapper.java

```diff
@@ -127,25 +129,19 @@ private void setActualDayParties(ActualHearingDayEntity actualHearingDayEntity,
                 }
             }
 
-            List<ActualIndividualDetails> individualDetailsList = new ArrayList<>();
-            List<ActualOrganisationDetails> organisationDetailsList = new ArrayList<>();
-            for (ActualAttendeeIndividualDetailEntity individualDetailEntity
-                : actualHearingPartyEntity.getActualAttendeeIndividualDetail()) {
-                actualDayParty.setPartyChannelSubType(individualDetailEntity.getPartyActualSubChannelType());
-                if (individualDetailEntity.getPartyOrganisationName() == null) {
-                    ActualIndividualDetails individualDetails = new ActualIndividualDetails();
-                    individualDetails.setFirstName(individualDetailEntity.getFirstName());
-                    individualDetails.setLastName(individualDetailEntity.getLastName());
-                    individualDetailsList.add(individualDetails);
-
-                } else {
-                    ActualOrganisationDetails organisationDetails = new ActualOrganisationDetails();
-                    organisationDetails.setName(individualDetailEntity.getPartyOrganisationName());
-                    organisationDetailsList.add(organisationDetails);
-                }
+            ActualIndividualDetails individualDetails = new ActualIndividualDetails();
+            ActualOrganisationDetails organisationDetails = new ActualOrganisationDetails();
```

Comment:
I don't think we should be creating these objects unless we actually need them. We can just create them in the `if`/`else` statement, only when required?

Reply:
changed